internal/enginecache/version.go

```diff
@@ -23,3 +32,105 @@ func flutterRequiredEngineVersion() string {
 
 	return versionMatch[1]
 }
+
+// CheckFoGoFlutterUpdate check the last 'go-flutter' timestamp we have cached
+// for the current project. If the last update comes back to more than X days,
+// fetch the last Github release semver. If the Github semver is more recent
+// than the current one, display the update notice.
+func CheckFoGoFlutterUpdate(goDirectoryPath string, currentTag string) {
```

Comment:
It feels a bit odd to have this function and `CurrentGoFlutterTag` in the enginecache internal package. Am I overlooking something, or could these be simply copied into their own package (e.g. versioncheck)?